scene/3d/label_3d.h

```diff
@@ -133,6 +133,9 @@ class Label3D : public GeometryInstance3D {
 	TextServer::StructuredTextParser st_parser = TextServer::STRUCTURED_TEXT_DEFAULT;
 	Array st_args;
 
+	bool auto_translate = true;
+	bool localize_numeral_system = true;
```

Comment:
I missed this one, should be removed too.

Reply:
You mean only `localize_numeral_system` right? isn't `auto_translate ` is necessary?

Reply:
Yes `localize_numeral_system` is now unused and should be removed. The other one is needed.